tests/services/test_metadata_backfill.py

```
@@ -108,17 +107,17 @@ def test_backfill_batch_commit_failure(self, db_session, tmp_path, file_inventor
         f1 = tmp_path / "test1.txt"
         f1.write_text("content1")
         inv1 = file_inventory_factory(path=str(f1), file_extension=None)
-        
+
         service = MetadataBackfillService(db_session)
-        
+
         # Mock commit to fail
         original_commit = db_session.commit
         def mock_commit():
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `mock_commit`

Add return type annotation: `NoReturn`

(ANN202)